website/content/docs/triggers.mdx

```diff
@@ -0,0 +1,179 @@
+---
+layout: docs
+page_title: Triggers
+description: |-
+  This page covers how to use Waypoint Triggers and their various use-cases.
+---
+
+# About Waypoint Triggers
+
+Waypoint 0.7 introduces a new core piece of functionality called Trigger URLs.
+Trigger URLs are pre-configured Waypoint lifecycle operations for execution
+inside Continuous Integration (CI) environments. Users can configure operations
+like deploying the latest build, running a status report on all applications in
+a project, running a Waypoint Up operation on all applications in a project,
+and more. Trigger URLs make it easy to run Waypoint operations inside CI without
+requiring the Waypoint CLI or any other Waypoint setup in your scripts. You will
+simply need grpcurl or curl to execute a Trigger URL
+
+Trigger URLs can be invoked directly inside an environment with a gRPC request
+that tells Waypoint to execute the pre-configured action. Trigger URLs
+configurations are project-scoped, where their actions can operate on every
+application defined in a project or target a single application inside a project.
+
+~> **NOTE**: With this first iteration in Waypoint 0.7, the Waypoint server will only
+support executing Triggers through gRPC. However, in a future version, we will
+support HTTP requests to initiate running Triggers, making Waypoint use in CI
+even more accessible than it is today. We recommend trying this feature through
+gRPC until the HTTP implementation has been released.
+
+## Waypoint Trigger CLI
+
+The Waypoint CLI comes with a subcommand that lets you configure and update
+Trigger URL configurations.
+
+Check out our CLI docs for the `waypoint trigger` command, such as
+[waypoint trigger create](/commands/trigger-create).
+
+## Pre-configuring a Trigger URL
+
+As an example, we can create a Trigger URL config that deploys the latest artifact
+to the produciton workspace. We'll need to figure the Trigger to use the
+right project, application, and workspace. Then we can specify which [lifecycle operation](docs/lifecycle)
+we want the trigger to execute when run. We can also include a `description` or
+`tags` for extra information for the user.
```

Comment:
```suggestion
As an example, you can create a Trigger URL config that deploys the latest artifact
to the production workspace. First, configure the Trigger to use the
right project, application, and workspace. Then, specify which [lifecycle operation](docs/lifecycle)
you want the trigger to execute when run. You can also include a `description` or
`tags` for extra information for the user.
```

Reply:
You go back and forth in the docs between `you` and `we` and I think we should be consistent. I've noted some suggested changes for this, but not everything.

Reply:
Thanks for the catch here! I merged your other suggestions but github doesn't like this multi-line suggestion so I'll make the changes locally